packages/ui/src/components/radio-button/radio-button.native.tsx

```diff
@@ -0,0 +1,26 @@
+import { Box } from '../box/box.native';
+
+interface RadioButtonProps {
+  isSelected: boolean;
+}
+export function RadioButton({ isSelected }: RadioButtonProps) {
```

Comment:
This looks good, but would've thought the UI lib has an actual button with radio like props and events, e.g. `onChange`

Reply:
The `RadioButton` isn't actually a button bc the entire row triggers the behavior, so maybe that is weird. If called a `...Button` maybe it should also have those properties so can be used on own?

Reply:
I added the `TouchableOpacity` here and passing the props so it can be used like a button on own. When triggered by an onPress on a row, I just used `disabled` prop or the two buttons conflicted.